### DISCUSSION TITLE
Add model zoo

### DISCUSSION THREAD

**Discussion by machineko** [google/flax#1315] [Ideas] [answered]
I see flax have been greatly upgraded in past few months.
I think model zoo would create healthy way to grow community even faster and let people do research easier thanks to pretrained models 😃

**Comment by marcvanzee** [comment, marked as answer]
Yes, we need more models! 😄

We deliberately avoid having a model zoo in our main repository, but we do have a list of curated [examples](https://github.com/google/flax/tree/master/examples). We aim for high quality and therefore would like to avoid ending up with a large number of models in our repository that don't work well or aren't maintained.

Currently you can find Flax models outside of our repo in the following places:

- We list community examples in readthedocs: [examples.html](https://flax.readthedocs.io/en/latest/examples.html). 
- We have a (tiny) model zoo at: https://github.com/google-research/google-research/tree/master/flax_models

If you are planning to create a model zoo yourself in your own repo, please let us know and we happily link to it!

**Reply by machineko** [reply]
I'll probably upload few models myself in near future 😄 
I still think model_zoo as outside repo will be good idea so it can get update for newest flax API and will be easier to search ( as for example in google research i can see only deprecated nn usage 🗡️ )

**Comment by andsteing** [comment]
See also #1320
